media-video/subliminal/subliminal-2.1.0-r1.ebuild

```diff
@@ -0,0 +1,100 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8,9} )
+PYTHON_REQ_USE='xml(+)'
+DISTUTILS_USE_SETUPTOOLS=rdepend
+
+inherit distutils-r1
+
+SRC_URI=""
+
+if [[ ${PV} == 9999 ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/Diaoul/${PN}.git"
+	EGIT_BRANCH="develop"
+else
+	SRC_URI="https://github.com/Diaoul/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+fi
+
+DESCRIPTION="Python library to search and download subtitles"
+HOMEPAGE="https://github.com/Diaoul/subliminal https://pypi.org/project/subliminal/"
+SRC_URI+=" test? ( mirror://sourceforge/matroska/test_files/matroska_test_w1_1.zip )"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
```

Comment:
```suggestion
```

Reply:
I did actually go to do these things, and then I had an error about test not being defined, so I couldn't use it in *DEPEND. I think it was just a bit late and I was doing something obviously wrong, because it's fine now.